core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/workflow/WorkflowResource.scala

```diff
@@ -86,6 +87,87 @@ object WorkflowResource {
 @Produces(Array(MediaType.APPLICATION_JSON))
 class WorkflowResource {
 
+  @GET
+  @Path("/ids")
```

Comment:
I think we need a better path than just `/ids`. Please add a comment on what this function returns.